CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 
 (Please put changes here)
+- Adding support for web identity provider, which enables IAM roles for Kubernetes service accounts.
```

Comment:
This pull does a bit more than just this — the Secret type is a big enough change to be listed here.